frontend/lib/ga/index.js

```diff
@@ -1,11 +1,15 @@
 // log the pageview with their URL
 export const pageview = (url) => {
-  window.gtag('config', process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS, {
-    page_path: url,
-  })
+  if (typeof window.gtag !== 'undefined') {
```

Comment:
I was getting a "window.gtag is not a function" error when I submitted the demo mode dataforms, so i added this check after reading this: https://stackoverflow.com/questions/62791299/typeerror-window-gtag-is-not-a-function

Reply:
sounds good

Reply:
I think we could also avoid this problem by removing google analytics from our packages in during https://github.com/pledge4future/WePledge/issues/131